src/azure-cli/azure/cli/command_modules/iotcentral/custom.py

```diff
@@ -20,7 +20,6 @@ def iotcentral_app_create(
 ):
     cli_ctx = cmd.cli_ctx
     _check_name_availability(client, app_name)
-    _check_subdomain_availability(client, subdomain)
```

Comment:
Hi, @kawayiboy Any idea to catch me up why remove this function `_check_subdomain_availability` ?